src/components/Header.jsx

```diff
@@ -32,7 +37,7 @@ const Header = () => {
   ];
 
   return (
-    <div>
+    <>
```

Comment:
If the div tag is removed, do we need empty ones?

Reply:
I believe so as we've got ` <Elements>
        <PaymentModal />
      </Elements>`

Reply:
Adjacent JSX elements must be wrapped in an enclosing tag. See example below:
```
const App = () => {
    return( 
       <div>
         <p>Hello</p>
         <p>World</p>
      </div>
     )
  }
```